### PR TITLE
Add back `informKyvernoPolicies` to policy generator

### PR DESCRIPTION
**Pull request by dhaiducek**
Revert generator removal:
- https://github.com/stolostron/rhacm-docs/pull/5613

ref: https://issues.redhat.com/browse/ACM-9450

### CODE REVIEW

File: governance/policy_generator.adoc [hunk 1]
```diff
@@ -286,6 +286,10 @@ Note that all the fields in the `policyDefaults` section except for `namespace`
 | Optional
 | Set `informGatekeeperPolicies` to false to use Gatekeeper manifests directly without defining it in a configuration policy. When the policy references a violated Gatekeeper policy manifest, an additional configuration policy is generated in order to receive policy violations in {product-title-short}. The default value is `true`.
 
+| `policyDefaults.informKyvernoPolicies`
+| Optional
+| When the policy references a Kyverno policy manifest, this determines if an additional configuration policy is generated to receive policy violations in {product-title-short}, when the Kyverno policy has been violated. The default value is `true`.
```
Comment (dockerymick) [inline governance/policy_generator.adoc]:
```suggestion
| When the policy references a Kyverno policy manifest, this determines if an additional configuration policy is generated to receive policy violations in {product-title-short}, when the Kyverno policy is violated. The default value is `true`.
```

**Review by dockerymick — APPROVED** [review]
Made a suggestion and committed. Thanks Dale. Approving this PR

**Review by gparvin — APPROVED** [review]
Thanks!

**Pull request merged**